app/cdap/components/PipelineDetails/ProfilesListView/ProfileCustomizePopover/ProfileCustomizeContent/index.js

```diff
@@ -83,6 +87,17 @@ export default class ProfileCustomizeContent extends PureComponent {
     this.latestValues = {};
   }
 
+  getDefaultPropertiesFromProvisionerDetails = (provisionerConfigGroups) => {
+    const properties = provisionerConfigGroups.flatMap((configGroup) => configGroup.properties);
+    const defaultProperties = {};
+    properties.forEach((property) => {
+      defaultProperties[property.name] = property['widget-attributes'].default
+        ? property['widget-attributes'].default
+        : '';
```

Comment:
Why not do `defaultProperties[property.name] = property['widget-attributes'].default`? Note that there are property types other than `text` with defaults (e.g. `workerDiskGB` has default `1000`).

Reply:
All the values are in String format 
<img width="368" alt="image" src="https://user-images.githubusercontent.com/98125204/234434213-eba2a461-724c-4300-bf9c-a1b48633d991.png">
And when empty, it's an empty string

Reply:
removed the unnecessary condition